pasqal_cloud/__init__.py

```diff
@@ -114,7 +114,9 @@ def create_batch(
         """
         if fetch_results:
             warn(
-                ("The parameter fetch_results has no effect and is deprecated."),
+                (
+                    "The parameter fetch_results will soon be deprecated. Please start using the `wait` parameter instead."
```

Comment:
Nit: If you're displaying a `DeprecationWarning`, it's already deprecated.

Maybe:
```
Argument `fetch_results` is deprecated and will be removed in a future version. Please use argument `wait` instead.
````